supervisor/updater.py

```diff
@@ -174,7 +174,7 @@ def channel(self, value: UpdateChannel):
         conditions=[JobCondition.INTERNET_SYSTEM],
         on_condition=UpdaterJobError,
         limit=JobExecutionLimit.THROTTLE_WAIT,
-        throttle_period=timedelta(seconds=30),
+        throttle_period=timedelta(seconds=600),
```

Comment:
This is a throttle, it's not actually checking every 30 seconds

Reply:
Yup, balloob is right, this is just prevents stalling the "check for update" button or hitting the update check service really fast doesn't hit upstream servers.

Reply:
Wrong way for handling it, as the supervisor itself only check every 6h.